File: VB/GridWithExpressions/ConditionExpressionEditor/ConditionExpressionEditorLogic.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports DevExpress.Data.ExpressionEditor
Imports DevExpress.Data

Namespace GridWithExpressions
	Friend Class ConditionExpressionEditorLogic
		Inherits ExpressionEditorLogicEx
		Public Sub New(ByVal editor As IExpressionEditor, ByVal columnInfo As IDataColumnInfo)
			MyBase.New(editor, columnInfo)

		End Sub

		Public Sub SetExpression(ByVal expression As String)
			ExpressionMemoEdit.Text = expression
		End Sub
	End Class
End Namespace

File: CS/GridWithExpressions/ConditionExpressionEditor/ConditionExpressionEditorLogic.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.Data.ExpressionEditor;
using DevExpress.Data;

namespace GridWithExpressions {
    class ConditionExpressionEditorLogic : ExpressionEditorLogicEx {
        public ConditionExpressionEditorLogic(IExpressionEditor editor, IDataColumnInfo columnInfo)
            : base(editor, columnInfo) {

        }

        public void SetExpression(string expression) {
            ExpressionMemoEdit.Text = expression;
        }
    }
}
